tools/cloud-build/daily-tests/tests/ml-a3-highgpu-slurm-cluster.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+test_name: a3h-cluster
+deployment_name: a3hc-{{ build }}
+slurm_cluster_name: "a3hc{{ build[0:4] }}"
+workspace: /workspace
+blueprint_yaml: "{{ workspace }}/examples/machine-learning/ml-slurm-a3-2-cluster.yaml"
+login_node: "{{ slurm_cluster_name }}-login-*"
+controller_node: "{{ slurm_cluster_name }}-controller"
+region: us-east4
+zone: us-east4-a
+network: default
+post_deploy_tests:
```

Comment:
I would like to see an nvidia_smi test. Fine to do in separate PR.

Reply:
Ack. Will document.